presto-orc/src/main/java/com/facebook/presto/orc/ByteArrayUtils.java

```diff
@@ -83,4 +86,21 @@ private static long longBytesToLong(long bytes)
     {
         return Long.reverseBytes(bytes) ^ Long.MIN_VALUE;
     }
+
+    public static long hash(byte[] bytes, int offset, int length)
```

Comment:
`hash` is a very generic name; do you expect to use this kind of hash somewhere else? if not, perhaps, move it into BytesValues class.